tests/app/dao/test_users_dao.py

```diff
@@ -148,11 +148,16 @@ def test_update_user_attribute(client, sample_user, user_attribute, user_value):
     assert getattr(sample_user, user_attribute) == user_value
 
 
+@freeze_time('2020-01-24T12:00:00')
 def test_update_user_password(notify_api, notify_db, notify_db_session, sample_user):
```

Comment:
This tests a user updating their password. Do we also need to test the case where a user is created for the first time that they have their last_validated_at set correctly?

Reply:
Cool, I will add a test like this 👍